app/views/world_wide_taxons/accordion.html.erb

```diff
@@ -38,7 +50,25 @@
             }
           %>
         <% end %>
-        <%= render 'govuk_publishing_components/components/accordion', items: items %>
+        <div data-module="gem-track-click gtm-click-tracking">
+          <div data-module="toggle-attribute">
+            <%
+              show_all_attributes = {
+                event_name: "select_content",
+                type: "accordion",
+                index: 0,
+                index_total: number_of_accordion_sections,
+                section: "n/a"
```

Comment:
`n/a` has been replaced with `null`, which doesn't need to be passed (see previous comment).